Rakefile

```diff
@@ -87,6 +87,22 @@ module Helpers
       platform.instance_eval { @version = $1 }
     end
 
+    if platform.os == 'java'
+      os_name = java.lang.System.get_property('os.name')
+      os_arch = java.lang.System.get_property('os.arch')
+      os = case os_name
+           when /linux/i then 'linux'
+           when /mac/i   then 'darwin'
+           else raise Error, "unsupported JRuby os.name: #{os_name.inspect}"
+           end
+      cpu = case os_arch
+            when 'amd64' then 'x86_64'
+            when 'aarch64' then os == 'darwin' ? 'arm64' : 'aarch64'
```

Comment:
I'm slightly confused with that. Accordingly to Wikipedia https://en.wikipedia.org/wiki/AArch64 aarch64 it's the same as arm64 (same set of instructions).

Same comes from ChatGPT

```
Yes, aarch64 and arm64 are essentially the same. Both terms refer to the 64-bit architecture developed by ARM Holdings:

aarch64: This is the official name given by ARM for their 64-bit architecture, especially used in technical documentation, compilers, and operating systems.
arm64: This is a more commonly used name and is often found in software packages, operating systems (like Linux distributions), and other contexts to denote the same architecture.
So, when you see aarch64 or arm64, they both refer to the 64-bit ARM architecture. They are typically used interchangeably.
```

Reply:
Yes it is the same but on [libddwaf release page](https://github.com/DataDog/libddwaf/releases), it uses arm64 for macOS and aarch64 for linux. I don't know why, but until it's fixed and we update to the fixed version (if there is ever one) we also have to use these terms accordingly

Reply:
Yes this is used to download and extract the libddwaf release, and similar changes have been made in lib_ddwaf.rb as the extracted folder will follow the same logic